src/forceUpdatedComponent.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+import { wrapDisplayName } from 'recompose'
+
+import { subscribe, unsubscribe } from './pubsub.js'
+
+/**
+ * Returns a HOC that triggers a re-render of itself whenever
+ * a subscription notification is received.
+ */
+export default function forceUpdatedComponent(InputComponent) {
+  class SubscribeUpdater extends PureComponent {
+    constructor(props) {
+      super(props)
+
+      this.state = {
+        lastNotification: null,
+      }
+    }
+
+    componentDidMount() {
+      subscribe(this.handleNotification.bind(this))
+    }
+
+    componentWillUnmount() {
+      unsubscribe(this.handleNotification)
+    }
+
+    handleNotification() {
```

Comment:
We could use a class property + bound function to avoid having to bind e.g.
```
handleNotification = () => {
   this.setState({
      lastNotification: Date.now(),
    })
}
```
and then we can remove the .bind()

Reply:
👍